### ISSUE TITLE
How to use nfs4j-client to create a Kerberos authentication mount and perform io operations with authentication information

### ISSUE THREAD

**Issue by hxedda** [dCache/nfs4j#107]
How to use nfs4j-client to create a Kerberos authentication mount and perform io operations with authentication information

**Comment by kofemann** [comment]
Hi @hxedda ,

unfortunately, the under laying RPC functionality provided by oncrpc4j doesn't lacks of the client side RPCSEC_GSS implementation.

**Comment by hxedda** [comment]
So the client currently cannot implement Kerberos authentication? Is there any plan to implement nfs4j-client in the future?

**Comment by hxedda** [comment]
> Hi @hxedda ,
> 
> unfortunately, the under laying RPC functionality provided by oncrpc4j doesn't lacks of the client side RPCSEC_GSS implementation.

So the client currently cannot implement Kerberos authentication? Is there any plan to implement nfs4j-client in the future?

**Comment by kofemann** [comment]
Yes, currently it's not possible to use RPCSEC_GSS in nfs client code. Obviously, we want our rpc layer to support it but there is no timeline when it it be implemented. Please open a request at https://github.com/dCache/oncrpc4j repo.
